docs/guides/js-waku/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: JavaScript Waku SDK
+---
+
+The [JavaScript Waku SDK](https://github.com/waku-org/js-waku) (`js-waku`) provides a TypeScript implementation of the [Waku protocol](/) designed for web browser environments. Developers can seamlessly integrate Waku functionalities into web applications, enabling efficient communication and collaboration among users using the `js-waku` SDK.
+
+## Installation
+
+Install the `js-waku` SDK using your preferred package manager:
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+<Tabs>
+<TabItem value="npm" label="npm">
+
+```shell
+npm install @waku/sdk
+```
+
+</TabItem>
+<TabItem value="yarn" label="Yarn">
+
+```shell
+yarn add @waku/sdk
+```
+
+</TabItem>
+</Tabs>
+
+You can also use the `js-waku` SDK via a CDN without installing it on your system:
+
+```js
+import * as waku from "https://unpkg.com/@waku/sdk@latest/bundle/index.js";
```

Comment:
This will be possible if we have the `/bundle/index.js` file in the root of the `@waku/sdk` package: https://www.npmjs.com/package/@waku/sdk?activeTab=code

The `crypto-js` package does something similar: https://www.npmjs.com/package/crypto-js?activeTab=code